Sources/Sentry/SentryFileManager.m

```diff
@@ -9,11 +9,13 @@
 NS_ASSUME_NONNULL_BEGIN
 
 NSInteger const defaultMaxEvents = 10;
+NSInteger const defaultMaxEnvelopes = 10;
```

Comment:
I believe it'd be way more. for Android, we default sessions (which is envelopes) to a 100

Reply:
Ideally we'll be writing multiple sessions per envelope soon which would mean possible more than 100. It's not a big deal but since this limit also affects (or will affect) crash events, we should probably be careful here.

If an average crash (uncompressed which is how we persist it today) is 200kb:

<img width="128" alt="image" src="https://user-images.githubusercontent.com/1633368/80486266-427b4880-8931-11ea-95d0-166618601511.png">

And we cap it to 100, we could possible take some 200mb of storage. Compressing would help but now we take more CPU when writing stuff to disk. 

Maybe we limit to 50 or some other magic number?

Reply:
Oh that'd be 20mb btw I'm bad at math

Reply:
I would just stick with `100` for now, because it's the same as on Android. 20MB could be fine on iOS, but on Android it could be a deal-breaker. Anyway the users can change it via the `SentryOptions`, but we don't have this on iOS yet. Before GA we should make sure users can change it, I guess. Are you fine with keeping `100`?

Reply:
wfm